### ISSUE TITLE
The difference between PAV and INS and DEL

### ISSUE THREAD

**Issue by shiyi-pan** [schneebergerlab/syri#265]
Hi, I am somewhat confused about the concepts of PAV (NOTAL) and INS or DEL.
In my opinion, INS (DEL) is a type of structural variation that clearly knows sequence information and sequence position, while PAV is a type of structural variation that only knows sequence information but not sequence position, as shown in the following figure.

DEL:
![image](https://github.com/user-attachments/assets/b7a59c97-dda0-4f00-a6b5-4977e043db2f)
PAV:
![image](https://github.com/user-attachments/assets/61ce82ba-ddb3-4909-849e-758b783a70ea)

I'm not sure if my understanding is correct. Looking forward to your reply.Thank you very much.

**Comment by mnshgl0110** [comment]
Yes, you are correct. For NOTAL, the corresponding position in the homologous chromosome is not obvious and is not reported. Please note that the syri.vcf only has NOTAL in the reference genome. NOTAL in the query genome cannot be written .VCF file (as that would require corresponding reference position) and are therefore only present in the syri.out file.
